src/styles/variables.scss

```diff
@@ -0,0 +1,25 @@
+$days-block: 100px;
+$block-padding-left: 100px;
+
+%full-flex-pack {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+%block-pack {
+  width: $days-block;
+  height: $days-block;
+}
+
+%flex {
+  display: flex;
+}
```

Comment:
does it make sense?

Reply:
No reason to create a placeholder for one prop.